ONSdigital/dp-frontend-dataset-controller
# Add test target and update depedencies so app builds

This pull request should fix the unit test and build run by Concourse, which are currently failing.

To test:

1) Check out this branch and make sure that `make build` and `make test` complete.
2) Check Concourse to make sure this branch is passing no there too.
